python/lsst/pex/config/comparison.py

```diff
@@ -156,16 +166,16 @@ def compareConfigs(name, c1, c2, shortcut=True, rtol=1e-8, atol=1e-8, output=Non
             return True
         else:
             if output is not None:
-                output(f"LHS is None for {name}")
+                output(f"{name}: None != {c2!r}.")
             return False
     else:
         if c2 is None:
             if output is not None:
-                output(f"RHS is None for {name}")
+                output(f"{name}: {c1!r} != None.")
             return False
     if type(c1) is not type(c2):
         if output is not None:
-            output(f"Config types do not match for {name}: {type(c1)} != {type(c2)}")
+            output(f"{name}: config types do not match; {type(c1).__qualname__} != {type(c2).__qualname__}.")
```

Comment:
Doesn't other code in pex_config use the `_typeStr()` function to obtain the full-qualified name?

Reply:
Oh, even better.  I was a little bummed that utils.introspection wasn't available, but I'm happy to use `_typeStr()`.

Reply:
We could depend on utils at this point, but not today.

Reply:
Yeah, and it's not worth it for just that, I think.